api/Request.json

```
@@ -990,6 +1050,10 @@
             "chrome_android": {
               "version_added": "46"
             },
+            "deno": {
+              "version_added": "1.0",
+              "notes": "In Deno the 'manual' redirect mode does not result in a opaqueredirect response, but a regular response."
```

Some style suggestions for this note. Might not be a bad idea to include a similar note on the `Request` constructor, too, since that's where the redirect mode must be set

```suggestion
              "notes": "The <code>\"manual\"</code> redirect mode does not result in a <code>"opaqueredirect"</code> response, but a regular response."
```

---

Rephrased, and added to `Request` constructor and `WindowOrWorkerGlobalScope.fetch`.